src/examples/accordions/timeline/TimelineCustomMedia.tsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Timeline } from '@zendeskgarden/react-accordions';
+import { Row, Col } from '@zendeskgarden/react-grid';
+import { Span } from '@zendeskgarden/react-typography';
+import { ReactComponent as LocationIcon } from '@zendeskgarden/svg-icons/src/12/location-stroke.svg';
+import { ReactComponent as RearrangeIcon } from '@zendeskgarden/svg-icons/src/12/rearrange-stroke.svg';
+import { ReactComponent as CartIcon } from '@zendeskgarden/svg-icons/src/12/shopping-cart-stroke.svg';
+import { ReactComponent as LeafIcon } from '@zendeskgarden/svg-icons/src/12/leaf-stroke.svg';
+import { StyledSpan } from './TimelineDefault';
```

Comment:
Oops, can you inline this? Otherwise there are errors launching the CodeSandbox.

Reply:
Also, goals for the website are for each example to support full copy/paste without needing access to local imports.

Reply:
Gotcha! Makes total sense!